Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -78,7 +80,8 @@ class LoginListViewController: UIViewController {
         super.viewDidLoad()
 
         let notificationCenter = NSNotificationCenter.defaultCenter()
-        notificationCenter.addObserver(self, selector: Selector("SELonProfileDidFinishSyncing"), name: NotificationProfileDidFinishSyncing, object: nil)
+        notificationCenter.addObserver(self, selector: Selector("SELreloadLogins"), name: NotificationProfileDidFinishSyncing, object: nil)
+        notificationCenter.addObserver(self, selector: Selector("SELreloadLogins"), name: NotificationDataLoginDidChange, object: nil)
```

Comment:
I don't think you need to listen to both. I think you need one and one only, which we might call `NotificationDataRemoteLoginChangesWereApplied`, and we issue that in the place you currently send `LoginDidChange`.